bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v2/ImageImpl.java

```diff
@@ -116,6 +119,34 @@ protected void initModel() {
         }
     }
 
+    private String getAssetName() {
```

Comment:
v2 extends v1, please call the method from v1 instead of duplicating it.

Reply:
fixed this